cluster/kubevirtci.sh

```diff
@@ -15,8 +15,24 @@
 export KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-'k8s-1.23'}
 export KUBEVIRTCI_TAG=$(curl -L -Ss https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirtci/latest)
 KUBEVIRTCI_PATH="${PWD}/_kubevirtci"
+KUBEVIRTCI_REPO='https://github.com/kubevirt/kubevirtci.git'
+
+function cluster::_get_repo() {
+    git --git-dir ${KUBEVIRTCI_PATH}/.git remote get-url origin
+}
+
+function cluster::_get_tag() {
+    git -C ${KUBEVIRTCI_PATH} describe --tags
+}
 
 function kubevirtci::install() {
+    # Remove cloned kubevirtci repository if it does not match the requested one
+    if [ -d ${KUBEVIRTCI_PATH} ]; then
+        if [ $(cluster::_get_repo) != ${KUBEVIRTCI_REPO} -o $(cluster::_get_tag) != ${KUBEVIRTCI_TAG} ]; then
```

Comment:
Nit: As far as I understand, the `||` operator is preferred, and I also think it's more readable than `-o`. Please consider to change it.

Reply:
why is preferable ?
i prefer to not change it, as it is the same thing we are doing on all the repos tbh
if you are fine with that
I know that "this what we do everywhere" is not an excuse, but as it is a nit i raised it